js/test/number/testnumfmt.js

```diff
@@ -4073,7 +4073,7 @@ module.exports.testnumfmt = {
     
         test.ok(fmt);
     
-        test.equal(fmt.format(100110.57), "ƒ 100.110,57");
```

Comment:
Most of these changes are changing from double quote to single quote, which does not make any semantic difference. I would not make these sort of unnecessary changes in a PR that is mixed with actual code changes from CLDR because it hides the real changes. Things like fixing whitespace or changing quotes or other such "clean up" type of changes should go into separate PRs.

Reply:
Yeah, I think I'd better closing this PR, then re-create PR to see changes clearly and I'll separate clean up PR if it's necessary.